crates/experimentation_platform/src/api/experiments/handlers.rs

```diff
@@ -550,9 +552,31 @@ async fn list_experiments(
 
     let query_builder = |filters: &ExpListFilters| {
         let mut builder = experiments::experiments.into_boxed();
-        if let Some(states) = filters.status.clone() {
+        if let Some(ref states) = filters.status {
             builder = builder.filter(experiments::status.eq_any(states.0.clone()));
         }
+        if let Some(ref experiment_name) = filters.experiment_name {
+            builder =
+                builder.filter(experiments::name.like(format!("%{}%", experiment_name)));
+        }
+        if let Some(ref context_search) = filters.context {
+            builder = builder.filter(
+                sql::<Bool>("context::text LIKE ")
```

Comment:
are we type casting here ? context

Reply:
Yes we are, this just lets me modify the context type so I can use LIKE on it. It's currently an experiment to see if this is a viable way to search through contexts